file_entity.module

```diff
@@ -257,7 +246,7 @@ function template_preprocess_file(&$variables) {
   $variables['file_url']  = $file->url('canonical');
   $label                  = $file->label();
   $variables['label']     = \Drupal\Component\Utility\SafeMarkup::checkPlain($label);
-  $variables['page']      = $view_mode == 'full' && file_entity_is_page($file);
+  $variables['page']      = $view_mode == 'full' && $file->isPage();
```

Comment:
That's an interesting one. It's a concept stolen from node (as many, many other things in this module) and it's one that I don't like. I'm wondering if we should just switch the label to be a configurable formatter through the base field definition and control with that if the title should be shown or not.  That's what I would like to do in core for node if I still could.